core/activity_pub.md

```diff
@@ -0,0 +1,40 @@
+# Activity Pub Compliance
+
+This document specifies the extensions to Fora required for a community to be compliant with the [Activity Pub](https://www.w3.org/TR/activitypub/) specification. Without this extension, the Fora kernel will not be capable of interacting with other Activity Pub implementations.
+
+## Introduction
+
+"it's just activity pub with governance"
+
+If we examine the Fora kernel through the lens of Activity Pub we find numerous similarities. Each user is a Client and each host is a Server, as defined by the Activity Pub specification. In fact, the only differences are a layer of governance settled on top of Activity Pub, along with additional properties between clients and servers, and servers and servers. The primary property added is the right to collective identification as a single community. Different servers with different sets of users can now collectively agree to be identified as the same community thereby sharing their state and community membership.
+
+The usefulness of this conceptual nesting is addressing the **right to individual exit** and the **right to collective exit**. 
+
+The former can be relatively easily addressed in Activity Pub by using flexible URI's such as DID's. This would give any user power to exit a community without losing it's account information (content may be lost, but followers/following lists would not).
```

Comment:
```suggestion
The former can be relatively easily addressed in Activity Pub by using flexible URI's such as DID's. This would give any user power to exit a community without losing its account information (content may have to be manually ported, but followers/following lists would automatically follow the user).
```